### ISSUE TITLE
Layout dimensions options

### ISSUE THREAD

**Issue by Forna73** [qgis/QGIS#34504]
After having looked for a way to create a layout with my custom dimensions, I discovered that:
- in the "layout tab" on the right there's no way to do that
- If you go in the layout menu on top, there's a "Page setup", but once "custom" is selected there's no way to input your dimensions"

To do that you must right click over the layout, in the main window and go for "Page properties".
I discovered this by googling on the web...

I think all these kind of options must be in the "Layout Tab" on the right as it was in the old "Print Composer" .
I can't imagine why these options have been left hidden.

**Comment by Forna73** [comment]
Any Answer or comment about this thing?
I think is much more useful and more logic...

**Comment by DelazJ** [comment]
Hi
If I may, I don't know about technical details but keep in mind that unlike QGIS 2, a layout in QGIS 3 can have multiple pages of different sizes and properties. Which page details would you display in the layout panel in case there are different pages types?
Maybe an "Update page properties" button next to the "Add page" button on the toolbar could help make it more accessible?
> I discovered this by googling on the web...

It's IMHO unfortunate to read many people complaining about how hard it had been for them to find this information while this is described in [QGIS documentation](https://docs.qgis.org/testing/en/docs/user_manual/print_composer/overview_composer.html#working-with-the-page-properties), and early after the change.  I know the docs is perfectible but we should sometimes give it a look...

**Comment by Forna73** [comment]
Hi!  Thank you for the reply...
I'm expirienceing now a big problem with printings... I use to create a pdf
with pdfcreator... wichever dimension is the page I obtain only a strange
format (a sort of tabloid)... no way to have my custom format...

Stefano

Il mar 21 apr 2020, 12:30 Harrissou Sant-anna <notifications@github.com> ha
scritto:

**Comment by DelazJ** [comment]
I've never used the printing options of the layout. Using the "Export to PDF" tool was enough to me. Without issues, often. Why not use it? And did you check the page size in page properties tab but also in pdfcreator setting?
Anyway, I think this is either another issue report (and you'll need to provide more information to devs so that they could try to replicate) or a help request (and you better use QGIS mailing lists - again details are welcome). We should keep this thread focused on page properties update tool's "discoverability".

**Comment by gioman** [comment]
> It's IMHO unfortunate to read many people complaining about how hard it had been for them to find this information while this is described in [QGIS documentation](https://docs.qgis.org/testing/en/docs/user_manual/print_composer/overview_composer.html#working-with-the-page-properties), and early after the change. I know the docs is perfectible but we should sometimes give it a look...

@DelazJ you totally are right, unfortunately the vast majority of users do not read the manuals (for some reason they tend to look for 3rd party tutorials/videos/etc first).

At the same time while I understand why now it is as it is, we must admit that as it was before it was more straightforward to find these configurations.

**Comment by Forna73** [comment]
Hi!
I used the manuals the first times I approached Qgis since 1.6 and further
2.0.... the print composer has been always straightforward so I apologize
if I miss something... I wished to let you know why I'm having difficulties
in this new layout thing... so you could refine the layout part.

Stefano

Il mar 21 apr 2020, 13:22 Giovanni Manghi <notifications@github.com> ha
scritto:

**Comment by DelazJ** [comment]
@Forna73 despite quoting your message, my comment was not targeting you. It's a general behavior I often see. Sorry if ever you felt offended, not my intention.
@gioman of course it was easier before and whatever argument we provide, this way of accessing the properties is clearly not obvious to us. This is why I was suggesting an update button. Browsing the issue reports, I found open #29033, #27843, #27698, #26237, that more or less point the same issue, with in some of them, suggestions of fixes. There are also closed reports like #25803. Worth a look, probably...

**Comment by Forna73** [comment]
Thanx man... don't worry I know what you were meaning... ;-)

Il giorno mar 21 apr 2020 alle ore 16:29 Harrissou Sant-anna <
notifications@github.com> ha scritto:

**Comment by Forna73** [comment]
I did some experiment today in 3.10.5... I discovered the in the Page SetUp menu not only once "custom" is selected there's no way to input your dimensions" but also that which ever format you select nothing happens. So this menu is totally broken to me.
I also can't understand the meaning about having a reference map in a layout....
What if I have different map windows of different sizes in a single page?
I think that has more sense having a page tab as per layout, object properties, atlas ecc... that's pop ups when page set up is clicked in the menu.

**Comment by DelazJ** [comment]
> I also can't understand the meaning about having a reference map in a layout....
What if I have different map windows of different sizes in a single page?

See https://docs.qgis.org/testing/en/docs/user_manual/print_composer/overview_composer.html#reference-map for what a reference map is designed for. Short: when expressions, variables or functions in the layout require a map, this is the map used by default if none is clearly indicated...
